src/reducers/auth.reducer.js

```diff
@@ -0,0 +1,23 @@
+import { userConstants } from "../constants/user.constants";
+
+let user = JSON.parse(localStorage.getItem("user"));
+const initialState = user ? { loggedIn: true, user } : {};
+
+export function authentication(state = initialState, action) {
+  switch (action.type) {
+    case userConstants.LOGIN_REQUEST:
+      return {
+        loggingIn: true,
```

Comment:
```suggestion
        loggedIn: true,
```

Reply:
The initial state is this:
```
let user = JSON.parse(localStorage.getItem("user"));
const initialState = user ? { loggedIn: true, user } : {};
```
And in `LOGIN_FAILURE` I return an empty object. Is that incorrect?

Reply:
Yes is perfect but I'm talking about the `loggingIn` not `loggedIn`

Reply:
Yes you're right!
Now I added a loader to use that `loggingIn`